file/sequence_file_reader.cc

```diff
@@ -25,19 +26,32 @@ namespace ROCKSDB_NAMESPACE {
 IOStatus SequentialFileReader::Create(
     const std::shared_ptr<FileSystem>& fs, const std::string& fname,
     const FileOptions& file_opts, std::unique_ptr<SequentialFileReader>* reader,
-    IODebugContext* dbg) {
+    IODebugContext* dbg, RateLimiter* rate_limiter) {
   std::unique_ptr<FSSequentialFile> file;
   IOStatus io_s = fs->NewSequentialFile(fname, file_opts, &file, dbg);
   if (io_s.ok()) {
-    reader->reset(new SequentialFileReader(std::move(file), fname));
+    reader->reset(new SequentialFileReader(std::move(file), fname, nullptr, {},
+                                           rate_limiter));
   }
   return io_s;
 }
 
-IOStatus SequentialFileReader::Read(size_t n, Slice* result, char* scratch) {
+IOStatus SequentialFileReader::Read(size_t n, Slice* result, char* scratch,
+                                    Env::IOPriority rate_limiter_priority) {
   IOStatus io_s;
+  uint64_t filesize;
+  IOOptions io_opts;
```

Comment:
`IOOptions io_opts;` does not seem to be used by your current impl

Reply:
Good catch..not sure why. This may be the reason? https://stackoverflow.com/questions/22112784/why-is-unused-variable-warning-not-reported-for-all-variables